cmd/auth/auth.go

```diff
@@ -26,6 +27,8 @@ var AuthCmd = &cobra.Command{
 	Long:  `Provide the necessary credentials to authenticate with your chosen backend.`,
 	Run: func(cmd *cobra.Command, args []string) {
 
+		var key = []byte{0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff, 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff, 0x00}
```

Comment:
I wonder if there could be another way to do this, like giving the user the option to set an encryption key or using something specific to them e.g. $WHOAMI etc

Reply:
Would change this then

Reply:
I have made the changes and added currentUser as the key

Reply:
@AlexsJones could you please review this change and merge the PR